### PR TITLE
feat: add lightweight styling to ComboBox

### PR DESCRIPTION
**Pull request by Arieldelossantos**
﻿GitHub Issue: # [[Lightweight Styling] ComboBox](https://github.com/unoplatform/uno.csharpmarkup/issues/247)

## PR Type

What kind of change does this PR introduce?

- Feature

## Description

add lightweight styling ComboBox

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested UWP
- [ ] Tested iOS
- [ ] Tested Android
- [ ] Tested WASM
- [ ] Tested MacOS
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml [hunk 1]
```diff
@@ -100,13 +100,11 @@
 			<StaticResource x:Key="ComboBoxPlaceHolderForegroundFocused" ResourceKey="OnSurfaceLowBrush" />
```
Comment (agneszitte) [inline src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:100]: @Arieldelossantos please the XAML changes separated in another PR so we can backport them properly without the C# changes (cc @kazo0, @Xiaoy312).

**Review by agneszitte — APPROVED** [review]

File: src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml
Comment (agneszitte) [inline src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:1]: @Arieldelossantos please have the XAML changes separated in another PR so we can backport them properly without the C# changes (cc @kazo0, @Xiaoy312).

Comment (Arieldelossantos) [reply src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:1]: done... here's the [PR]( https://github.com/unoplatform/Uno.Themes/pull/1073)

**Review by agneszitte — APPROVED** [review]

File: src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml
Comment (agneszitte) [reply src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml:1]: @Arieldelossantos merged thank you

**Pull request merged**